### ISSUE TITLE
Implement Payload Data Type constraints feature for type checking at runtime

### ISSUE THREAD

**Issue by isocroft** [isocroft/Radixx#2]
```js

/* semantic checking of type(s) */

// Examples:

Radixx.Payload.type.number 
Radixx.Payload.type.string
Radixx.Payload.type.array
Radixx.Payload.type.boolean

```

Using the {setPayloadDefinition} for action creator data

**Comment by synergixe** [comment]
This issue has been implemented completely and added into version 0.1.1 of Radixx

Closing this issue...

**Comment by isocroft** [comment]
This Feature has been implemented...

So closing this issue!
